examples/README.md

```diff
@@ -88,6 +93,10 @@ In addition, integration modules are available for the following libraries, prov
 
 * [Optimizing on Kubernetes](./kubernetes/README.md)
 
+### Examples of Reinforcement Learning
+
+* [Optimizating Hyperparameters for Stable-Baslines Agent](./rl/sb3_simple.py)
```

Comment:
Fix typo
```suggestion
* [Optimization of hyperparameters for Stable-Baslines Agent](./rl/sb3_simple.py)
```

Reply:
Thank you for pointing that out, I'll change that.